src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -64,16 +88,20 @@ class IteratorExpr:
     Args:
         field: The field this iterator operates on.
         dimensions: Field domain represented as a sorted list of dimensions.
-                    In order to dereference an element in the field, we need index values
```

Comment:
I would rephrase this and combine the sentences.

Reply:
Done.